wsclient/types.go

```diff
@@ -0,0 +1,60 @@
+package wsclient
+
+const (
+	BASE_URL          = "wss://api.tanx.fi"
```

Comment:
can we have a testnet mode to the library, you will just replace this url when in tn mode

Reply:
Actually I will provide methods to configure the client

Reply:
still i feel an option to quickly change mode to testnet would be good to have